src/pages/lessons/[slug].tsx

```diff
@@ -49,44 +50,42 @@ const lessonQuery = /* GraphQL */ `
 
 const fetcher = (url: RequestInfo) => fetch(url).then((r) => r.json())
 
+const getNextUpData = (url: string) => {
+  const {data} = useSWR(url, fetcher)
+  return data
+}
+
```

Comment:
Make this request as separate function so that we can get data with one request and use it for few places (there are two cases at least - `data.list.lessons` for the `NextUp` component and `data.next_lesson` for the `PlayerControls` component)

Reply:
I think this should probably be named like a React Hook useCollectionData, but looks good really. That’s just a convention